package.json

```diff
@@ -59,29 +59,29 @@
-    "test_local_unit_consumer": "npm run pretest && npx ava --timeout=20m --verbose --concurrency=1 build/test/test/source/test.js -- UNIT-TESTS --retry=false --pool-size=1 --debug CONSUMER-MOCK",
-    "test_local_unit_enterprise": "npm run pretest && npx ava --timeout=20m --verbose --concurrency=1 build/test/test/source/test.js -- UNIT-TESTS --retry=false --pool-size=1 --debug ENTERPRISE-MOCK",
+    "test_local_unit_consumer": "npm run pretest && npx ava --timeout=20m --verbose --concurrency=10 build/test/test/source/test.js -- UNIT-TESTS --retry=false --debug CONSUMER-MOCK",
+    "test_local_unit_enterprise": "npm run pretest && npx ava --timeout=20m --verbose --concurrency=10 build/test/test/source/test.js -- UNIT-TESTS --retry=false --pool-size=1 --debug ENTERPRISE-MOCK",
     "test_local_chrome_consumer_mock_headless": "xvfb-run npm run test_local_chrome_consumer_mock",
     "test_stylelint": "stylelint extension/css/cryptup.css extension/css/settings.css extension/css/webmail.css && stylelint extension/**/*.htm --custom-syntax postcss-html",
     "test_eslint": "eslint --ext ts extension test tooling",
     "test_patterns": "node build/test/test/source/patterns.js",
     "test_async_stack": "node build/test/test/source/async-stack.js",
-    "test_buf": "npx ava --timeout=20m --verbose --concurrency=1 build/test/test/source/buf.js",
-    "test_ci_chrome_consumer_live_gmail": "npx ava --timeout=20m --verbose --concurrency=1 build/test/test/source/test.js -- CONSUMER-LIVE-GMAIL STANDARD-GROUP",
-    "test_ci_chrome_consumer": "npx ava --timeout=20m --verbose --concurrency=1 build/test/test/source/test.js -- CONSUMER-MOCK STANDARD-GROUP",
-    "test_ci_chrome_enterprise": "npx ava --timeout=20m --verbose --concurrency=1 build/test/test/source/test.js -- ENTERPRISE-MOCK STANDARD-GROUP",
-    "test_ci_chrome_consumer_flaky": "npx ava --timeout=20m --verbose --concurrency=1 build/test/test/source/test.js -- CONSUMER-MOCK FLAKY-GROUP",
-    "dev_start_gmail_mock_api": "./build.sh && cd ./conf && node ../build/tooling/tsc-compiler --project tsconfig.test.json && cd .. && node ./build/test/test/source/mock.js",
+    "test_buf": "npx ava --timeout=20m --verbose --concurrency=10 build/test/test/source/buf.js",
+    "test_ci_chrome_consumer_live_gmail": "npx ava --timeout=20m --verbose --concurrency=10 build/test/test/source/test.js -- CONSUMER-LIVE-GMAIL STANDARD-GROUP",
+    "test_ci_chrome_consumer": "npx ava --timeout=20m --verbose --concurrency=10 build/test/test/source/test.js -- CONSUMER-MOCK STANDARD-GROUP",
+    "test_ci_chrome_enterprise": "npx ava --timeout=20m --verbose --concurrency=10 build/test/test/source/test.js -- ENTERPRISE-MOCK STANDARD-GROUP",
+    "test_ci_chrome_consumer_flaky": "npx ava --timeout=20m --verbose --concurrency=10 build/test/test/source/test.js -- CONSUMER-MOCK FLAKY-GROUP",
```

Comment:
What is the purpose of the "flaky" group? There are resource-demanding or tests with an unstable performance. This group is meant for tests to be run consequently, one at a time

Reply:
Some tests are inexplicably resource-demanding. When run in parallel with other tests, what can happen is that either that test will time out, or other tests will fail while the resource-intensive test hogs up resources.

Other reason that a test may end up in "flaky" section is if it needs to be executed serially for some other reason.

Generally speaking, these tests tend to cause reliability trouble  when executed in parallel, for one reason or another.

Of course, they could be debugged and fixed one by one and moved, possibly, if we had the time or willingness.

Reply:
Does this mean we should set `--concurrency=1` instead of `--concurrency=10`?

Reply:
Yes, we can change it back to `--concurrency=1` as browser pool size is automatically set to 1 for `FLAKY-GROUP` - https://github.com/FlowCrypt/flowcrypt-browser/blob/master/test/source/util/index.ts#L31

Reply:
Shouldn't browser pool and concurrency always be exactly equal? Or is there a reason to have these two numbers diverge? I'd imagine, we could remove special code to choose that `FLAKY-GROUP` has 1 in the pool, and instead follow whatever the provided `--concurrency` ?

Reply:
Probably it can be simplified, as currently `--pool-size` parameter is used for determining `oneIfNotPooled`, which then also used for setting `ATTEMPTS` constant - https://github.com/FlowCrypt/flowcrypt-browser/blob/master/test/source/test.ts#L40

Reply:
https://github.com/FlowCrypt/flowcrypt-browser/issues/4955